pkg/manifest/config/validate.go

```diff
@@ -20,7 +20,10 @@ func (m *Manifest) Validate() error {
 	vv := &validateVisitor{errs: []error{}, envNames: map[string]bool{}, appNames: map[string]bool{}, serviceNames: map[string]bool{}}
 
 	m.Walk(vv)
-	return multierror.Join(vv.errs)
+	if len(vv.errs) > 0 {
```

Comment:
This implies a missing test?

Reply:
Checking for a positive validation was missing.I've updated the test case